src/App.svelte

```diff
@@ -0,0 +1,494 @@
+<script>
+// import { onMount } from "svelte";
+let value = 0;
+let counter = 1;
+let themeValue = 1;
+let storage = [0,0];
```

Comment:
It seems you don't need an array here since you never used storage[1]. You can use number variable instead.

Reply:
changed